src/test/crimson/seastore/test_seastore.cc

```diff
@@ -451,7 +470,7 @@ TEST_F(seastore_test_t, attr)
 {
   run_async([this] {
     auto& test_obj = get_object(make_oid(0));
-
+  {
```

Comment:
I mentioned before 4 cases that need to be tested specifically, can you add comments indicating where they are?

Reply:
done.